firstvoices/backend/models/characters.py

```diff
@@ -220,27 +220,6 @@ class Meta:
     # JSON representation of a g2p mapping from confusable characters to canonical characters
     input_to_canonical_map = models.JSONField(default=list)
 
-    @property
-    def base_characters(self):
-        """
-        Characters for the site in sort order.
-        """
-        return Character.objects.filter(site=self.site).order_by("sort_order")
-
-    @property
-    def variant_characters(self):
-        """
-        All variant characters for the site.
-        """
-        return CharacterVariant.objects.filter(site=self.site)
-
-    @property
-    def ignorable_characters(self):
-        """
-        Ignorable characters for the site.
-        """
-        return IgnoredCharacter.objects.filter(site=self.site)
-
```

Comment:
Any reason to remove these properties? Seems like they could still be a useful way to get the default values in the other functions, if the context/parameters are not missing

Reply:
I have the same question, do they have to be removed to reduce the queries?